### PR TITLE
Dev/error handling ts restart

### PR DESCRIPTION
**Pull request by chunderstruck**
Added Error Handling for Base64 Submissions in the event that incorrect parameters are provided.

Added functions to stop and start timesketch.

### CODE REVIEW

File: scripts/rc.py [hunk 1]
```diff
@@ -38,6 +38,16 @@ def add_ts_parsers(subparsers):
                         metavar="timesketch_name",
                         help="Delete the Base64 encoded timesketch name provided")
 
+    group.add_argument('--start',
```
Comment (rough007) [inline scripts/rc.py]: This is already in the allowed --service <start/restart/stop> timesketch function. So these are duplicative.  Or am I missing something?

File: scripts/rc.py [hunk 2]
```diff
@@ -38,6 +38,16 @@ def add_ts_parsers(subparsers):
...
+                        nargs='?',
+                        const="127.0.0.1",
+                        metavar="listener",
+                        help="Starts Timesketch Service")
+
+    group.add_argument('--stop',
```
Comment (rough007) [inline scripts/rc.py]: This is already in the allowed --service <start/restart/stop> timesketch function. So these are duplicative.  Or am I missing something?

**Comment by chunderstruck** [comment]
I thought I had tried that ... apparently not.  I thought I checked services.

**Comment by chunderstruck** [comment]
Suppose could use the same functions to activate the service...

File: scripts/rc.py [hunk 3]
```diff
@@ -125,6 +139,19 @@ def delete_ts(ts,enc_name):
...
+    logger.debug(listener)
+    ts_parameters = [ts, "runserver", "-h", listener] 
+    logger.debug(ts_parameters)
+    cmd = subprocess.Popen(ts_parameters, stdout=subprocess.PIPE)
+    logger.info("Timesketch Running pid: {} - listening on {}".format(cmd.pid, listener))
+
+def ts_stop_server():
+    logger.debug("Attempting to stop timesketch server")
+    logger.info("Stopping TimeSketch Service")
+    cmd = subprocess.call("killall tsctl", shell=True, stdout=subprocess.PIPE)
```
Comment (jdf400-rapid7) [inline scripts/rc.py]: For all subprocess.call calls can we add a bit to check if the output was non 0 and print it out if it was

**Pull request closed without merging**